app/src/main/res/layout/activity_norwegian_authentication.xml

```diff
@@ -8,7 +8,7 @@
 
     <!-- TODO: Text Key -->
     <androidx.appcompat.widget.Toolbar
-        android:id="@+id/hedvigToolbar"
+        android:id="@+id/toolbar"
```

Comment:
Not related at all to the PR, but now that we found this TODO in this file, I really want to fix it 😅

Reply:
Yes im talking to zak about this atm